packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -480,6 +482,14 @@ function SelectedTransactionsButton({
     };
   }, [selectedItems]);
 
+  let ambiguousDuplication = useMemo(() => {
+    let transactions = [...selectedItems].map(id => getTransaction(id));
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    let transactions = selectedItems.map(id => getTransaction(id));
```

Reply:
@MatissJanis, had to add this back, selectedItems is a Set and can't use .map